uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -532,10 +535,7 @@ sub determineScannerID {
 
 =head3 get_acqusitions($study_dir, \@acquisitions)
 
-Greps the list of acquisitions in the study directory from array
-C<@acquisitions>. ?????????????????????????????????????
-
-INPUT: study directory, array of acquisitions ??????????????????
+UNUSED FUNCTION TO BE CLEANED UP IN ANOTHER PULL REQUEST
```

Comment:
maybe just say unused here and in the TODO section, we can say removal of unused utilities/methods (such as get_acquisitions) perhaps?

Reply:
otherwise, tested and everything else looks good.